index.html

```diff
@@ -0,0 +1,211 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Home Bulza Festival</title>
+    <script src="https://kit.fontawesome.com/9f089b925d.js" crossorigin="anonymous"></script>
+</head>
+
+<body>
+    <!--Social media icons-->
+    <div class="navbar-d hidden">
+        <div><i class="fa fa-facebook"></i></div>
+        <div><i class="fa fa-twitter"></i></div>
+        <div><i class="fa fa-soundcloud"></i></div>
+        <div><i class="fa fa-youtube-square"></i></div>
+    </div>
```

Comment:
- [ ] [OPTIONAL] For this section you could use a `ul` tag and your inner divs become `li` of the `ul`  it could be something like this :point_down: 
    ```HTML 
                                    <ul>
                                         <li><i class="fa fa-facebook"></i></li>
                                         <li><i class="fa fa-twitter"></i></li>
                                   </ul>
     ```